### PR TITLE
Bartosz slowik/config

### PR DESCRIPTION
**Pull request by Bartosz-Slowik**
Adding the config to repo will help new users to run the program.
Removed config from gitignore.

### CODE REVIEW

**Review by wiktorwieclaw — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
 # CanSat
+## Prerequisites
+ 1. You will need **libusb** <br>
+ check out [cargo-embed](https://crates.io/crates/cargo-embed) for instructions
+ 2. Add your board target:
+```
+rustup target add thumbv7em-none-eabihf
+```
+
```
Comment (wiktorwieclaw) [inline README.md:9]: I guess this change was not supposed to be in this PR considering the existence of https://github.com/grupacosmo/cansat/pull/7

File: cansat-stm32f446/.cargo/config.toml [hunk 2]
```diff
@@ -3,5 +3,6 @@ rustflags = [
   "-C", "link-arg=-Tlink.x",
 ]
 
+
```
Comment (wiktorwieclaw) [inline cansat-stm32f446/.cargo/config.toml:6]: Try not to add random lines like this :P

**Review by wiktorwieclaw — CHANGES_REQUESTED** [review]
See https://github.com/grupacosmo/cansat/issues/8#issuecomment-1354050455

**Comment by Bartosz-Slowik** [comment]
Fixed requested issues

**Review by wiktorwieclaw — APPROVED** [review]

**Pull request merged**